calico/datamodel_v1.py

```diff
@@ -67,7 +67,7 @@
     r'^' + PROFILE_DIR + r'/(?P<profile_id>[^/]+)/tags')
 # Regex to match endpoints, captures "hostname" and "endpoint_id".
 ENDPOINT_KEY_RE = re.compile(
-    r'^' + HOST_DIR +
+    r'^(?:' + HOST_DIR + r'|' + FELIX_STATUS_DIR + r')'
```

Comment:
Worth extending the comment to say that this works either for endpoints under the host, or endpoints under the status directory? Not 100% sure since it's kind of obvious.